src/EIP7702Proxy.sol

```diff
@@ -84,43 +89,43 @@ contract EIP7702Proxy is Proxy {
     /// @param args The initialization arguments for the implementation
     /// @param signature The signature authorizing initialization
     /// @param crossChainReplayable use a chain-agnostic or chain-specific hash
-    function initialize(
-        bytes calldata args,
-        bytes calldata signature,
-        bool crossChainReplayable
-    ) external {
-        // Construct hash using typehash to prevent signature collisions
-        bytes32 initHash = keccak256(
-            abi.encode(
-                _INITIALIZATION_TYPEHASH,
-                crossChainReplayable ? 0 : block.chainid,
-                _PROXY,
-                NONCE_TRACKER.useNonce(),
-                keccak256(args)
-            )
-        );
-
-        // Verify signature is from this address (the EOA)
-        address signer = ECDSA.recover(initHash, signature);
-        if (signer != address(this)) revert InvalidSignature();
-
-        // Initialize the implementation
-        ERC1967Utils.upgradeToAndCall(
-            INITIAL_IMPLEMENTATION,
-            abi.encodePacked(GUARDED_INITIALIZER, args)
-        );
-    }
-
-    /// @notice Resets the ERC-1967 implementation slot after signature verification, allowing the account to
-    ///         correct the implementation address if it's ever changed by an unknown delegate or implementation.
-    ///
-    /// @dev Signature must be from the EOA's address that is 7702-delegating to this proxy
-    ///
+    // function initialize(
```

Comment:
just commented this out for now so we can look at it but presumably we can get rid of the specific initialize function in this new flow